sdk/servicefabricmanagedclusters/Azure.ResourceManager.ServiceFabricManagedClusters/CHANGELOG.md

```diff
@@ -10,12 +10,26 @@
 
 ### Other Changes
 
-## 1.1.0-beta.2 (2023-06-21)
+## 1.1.0-beta.2 (2023-06-21)### Features Added
```

Comment:
Please remove the "### Features Added" in this line

Reply:
updated